docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -7,4 +7,108 @@
 :keywords: {ml-init}, {stack}, {nlp}, ELSER
 :description: ELSER is a learned sparse ranking model trained by Elastic.
 
-[TBD]
\ No newline at end of file
+experimental[]
+
+Elastic Learned Sparse Encoder - or ELSER - is a text expansion model that uses 
+sparse vector representation and trained by Elastic . As a retrieval model, 
```

Comment:
```suggestion
Elastic Learned Sparse Encoder - or ELSER - is a representational model trained by Elastic that creates a
sparse vector representation  . As a retrieval model, 
```

Text expansion the _user friendly_ name we have chosen for the search query but it isn't a common term that people will recognise. I like @qherreros description of expanding text, maybe use the 'text expansion' after that